modular_skyrat/modules/events/code/event_overrides.dm

```diff
@@ -57,8 +57,8 @@
  * Upped to ensure lowpop steamroll does not happen
  */
 /datum/round_event_control/spider_infestation
-	// min_players = 70
-	max_occurrences = 0
+	min_players = 40 //BUBBERS EDIT CHANGE, ORIGINAL: //min_players = 70
```

Comment:
Leave these as is, we had this once(via the dynamic ruleset) and it was pretty disastrous.